back/engines/commercial/project_folders/app/models/project_folders/patches/user.rb

```diff
@@ -4,6 +4,7 @@ module ProjectFolders
   module Patches
     module User
```

Comment:
As a result of moving a few things back into the core, project-folder patch was a bit simplified.